charts/kubefed/values.yaml

```diff
@@ -27,6 +27,7 @@ controllermanager:
     FederatedIngress:
   
   controller:
+    annotations: {}
```

Comment:
Reason for this? Does omitting this break something?

Reply:
would be nice to inject annotations to the deployments.